src/Microsoft.AspNetCore.Mvc.RazorPages/Infrastructure/PageLoaderResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure
+{
+    public struct PageLoaderResult
```

Comment:
Can you explain c to me? Why isn't `Pages/_ViewImports.cshtml` included by `**/*.cshtml`?